app/views/posts/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="topnav">
+  <%= link_to image_tag("hacker-logo.png", action: 'index', size: "22x22", alt: "Logo")%>
+  <span class="bold"><%= link_to('Hacker News', action: 'index' , class: "text-center") %></span> |
+  <%= link_to('New', action: 'index' , class: "text-center") %> |
+  <%= link_to('Submit Post', action: 'new' , class: "text-center") %>
+</div>
+
+<div class="posts">
+  <% post_num = 1 %>
+  <% @posts.each do |p| %>
```

Comment:
Maybe  [`each_with_index`](https://ruby-doc.org/core-2.5.1/Enumerable.html#method-i-each_with_index) instead.

Reply:
But actually a better answer might be to use [rails rendering helpers](https://guides.rubyonrails.org/layouts_and_rendering.html#rendering-collections). Splitting this row out into a partial is probably a good idea anyways, you'll probably be adding more functionality / markup to it later on.

Reply:
Thanks, I'll look into the rendering helpers

Reply:
+1

Reply:
I got it working with a partial, pretty cool